app/main/util/date_validator.py

```diff
@@ -0,0 +1,334 @@
+from datetime import date, datetime
+
+from flask import current_app
+
+db_date_format = "%Y-%m-%d"
+python_date_format = "%d/%m/%Y"
+default_start_date = None
+check_future_date = True
+
+
+def validate_date_range(date_range):
```

Comment:
Could we now remove this validation function which is used in the query building if we validate the request args now?